src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+
+type FooterProps = {
+  todos: Todo[];
+  status: string;
+  setStatus: (status: string) => void;
+};
+export const Footer: React.FC<FooterProps> = ({ todos, status, setStatus }) => {
+  const activeTodoCount = todos.filter(todo => !todo.completed).length;
+  const completedTodoCount = todos.filter(todo => todo.completed).length;
```

Comment:
As I see, you can use reduce and return object with activeTodoCount and completedTodoCount. 
In this case you have one array iteration. Not critical, alternative variant.